schemaregistry/schemaregistry.md

```diff
@@ -405,3 +497,196 @@ The returned ´Content-Type´ is the same that was passed when the schema versio
+services that cannot host their own service registry endpoints.
+
+A star replication topology uses hubs central hubs into which all sources
+contribute schemas and to which all targets subscribe. If the hub is a centrally
+governed registry, it might not accept inbound replications.
+
+### 4.3 Push replication
+
+In an imperative "push" replication model, a registry might push any updates of
+its state to one of more configured target registry by forwarding all changes to
+the respective schema registry API calls on that target registry.
+
+The pushing registry might be configured to copy changes of the full registry, of
+a specific schema group, or just of a specific schema.
+
+This replication model uses the registry API as defined above on the target
+registry and the configuration of what aspects of the registry are pushed are
+implementation specific.
+
+### 4.4 Pull replication
+
+In an imperative "pull" replication model, a target registry will once or periodically
+traverse the source registry contents and apply differences between the source
+state and its own state.
+
+The pulling registry might be configured to copy changes of the full registry, of
+a specific schema group, or just of a specific schema.
+
+This replication model uses the registry API as defined above on the source
+registry and the configuration of what aspects of the registry are pulled are
+implementation specific.
+
+### 4.5. Event-driven replication
+
+Event-driven replication uses CloudEvents to notify interested parties of
+changes in the source registry such that those parties can
+[pull](#44-pull-replication) changes immediately and as they become available.
+
+Each registry SHOULD offer a [Subscription API](../subscriptions-api.md)
+endpoint, either directly or using some middleware, to allow interested parties
+to subscribe to these change events.
+
+The subscription's `source` MUST be the root of the schema registry. A [prefix
+filter](../subscriptions-api.md#prefix-filter-dialect) on the `subject`
+attribute MAY be used to scope the subscription to a particular schema group or
+schema.
+
+The subscribing party registers a notification sink of its choosing.
+
+```HTTP
+POST https://source.example.com/subscriptions HTTP/1.1
+Content-Type: application/cloudevents-subscription+json
+
+{
+  "source": "https://source.example.com",
+  "protocol" : "HTTPS",
+  "sink" : "https://target.example.com/notifications",
+  "filter" : {
+    "prefix" : {
+       "subject" : "/schemagroups/5"
+    }
+  }
+}
+```
+
+#### io.cloudevents.schemaregistry.updated.v1
+
+This event notifies the subscriber of an element of the registry having been
+created or updated. Because schema documents might be very large, and because
+schema information might be subject to special authorization since it might
+disclose trade secrets, the event carries no data, but requires the consumer to
+fetch the indicated change from the registry.
+
+* **source**: Base URI of the schema registry
```

Comment:
The Base URI is defined as the authority by default.
In the case of replication is it the original authority or the hub authority in `source`?

Reply:
I hopefully managed to clarify that in 4.3 and 4.4.  Here, it is the BaseURI indeed. It is the path to the "local copy" in the registry raising the event.